JAVA2/.gitignore

```diff
@@ -0,0 +1,3 @@
+out/
```

Comment:
Out directory also got pushed. Make sure that gitignore is working properly before committing the changes.